CMakeLists.txt

```diff
@@ -212,6 +212,7 @@ if(OXENMQ_INSTALL)
         oxenmq/bt_serialize.h
         oxenmq/bt_value.h
         oxenmq/connections.h
+        oxenmq/formattable.h
```

Comment:
Should rename this to `fmt.h` to make it clearer that it's for (optional) fmt library support